app/src/main/res/values/strings.xml

```diff
@@ -21,17 +21,17 @@
     <string name="signed_in">You are signed in</string>
     <string name="not_signed_in">Welcome</string>
     <string name="sign_in">Sign in</string>
-    <string name="you_are_not_signed_in">You are not signed in</string>
-    <string name="sign_in_error">Unable to Load Sign-In Info</string>
-    <string name="sign_in_error_desc">There was an error reading the last login statue. Please clean the app storage and launch again. This message needs to be updated by ux</string>
-    <string name="no_internet">No Internet</string>
-    <string name="no_internet_desc">No Internet</string>
+    <string name="you_are_not_signed_in">Please sign in to continue.</string>
+    <string name="sign_in_error">Signed Out</string>
+    <string name="sign_in_error_desc">You have been signed out, likely due to an application error or prolonged inactivity.\nSign in to continue.</string>
```

Comment:
I'm a bit confused why we have these two strings same as line 35 & 36 but with different names.

Reply:
Internally they represent different errors but to make it easier for the user they have the same description.

Reply:
Then maybe we can use a more generic name and combine the two?

Reply:
I think it's appropriate to keep them represented separately internally, in this case.